rabbit-escape-ui-android/app/src/main/java/rabbitescape/ui/android/AndroidCanvas.java

```diff
@@ -40,6 +43,24 @@ public void drawLine(
         float startX, float startY, float stopX, float stopY, AndroidPaint paint
     )
     {
-        this.canvas.drawLine( startX, startY, stopX, stopY, paint.paint );
+        this.canvas.drawLine(startX, startY, stopX, stopY, paint.paint);
+    }
+
+    @Override
+    public void drawFilledPoly( int[] xs, int[] ys, AndroidPaint paint )
```

Comment:
so I should 
- make a Path interface in render with moveTo() lineTo() and close(), ignoring the 30 or so other methods in android.graphics.Path.
- make a drawPath method for swing.